projects/ion/src/lib/pagination/pagination.component.html

```diff
@@ -2,7 +2,7 @@
   <ion-button
     rightSideIcon="true"
     *ngIf="allowChangeQtdItems"
-    [label]="optionsPage[0].label"
+    [label]="getSelectedItemsPerPageLabel(optionsPage)"
```

Comment:
It is not interesting to update a label using functions directly in the html, because every time there is some change on the screen, the function will be called, even if what is changing has nothing to do with the label. I suggest creating a variable and updating through it